style.css

```diff
@@ -1205,7 +1205,105 @@
     margin-left: 5px;
   }
 
-  .projects {
+  .contact-form {
     display: flex;
+    margin-top: 50px;
+    margin-bottom: 50px;
+    border-top-left-radius: 80px 80px;
+    padding: 10px 24px;
+    flex-direction: column;
+    background: #6070ff;
+    align-items: center;
+    background-image: url(assets/images/contact-form-bg/contact\ form\ shapes-mobile.svg);
+    background-repeat: no-repeat;
+    background-position-x: 195px;
+  }
+
+  .contact-form h2 {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    margin-top: 75px;
+    color: #fff;
+  }
+
+  .contact-form p {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+  }
+
+  .contact-form form {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    gap: 10px;
+  }
+
+  input[type="text"] {
+    height: 48px;
+    padding-left: 15px;
+    border: none;
+    border-radius: 8px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 20px;
+  }
+
+  input[type="email"] {
+    height: 48px;
+    padding-left: 15px;
+    border: none;
+    border-radius: 8px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 20px;
+  }
+
+  .contact-form textarea {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    padding-left: 15px;
+    padding-top: 15px;
+    line-height: 24px;
+    resize: none;
+    color: #b3bac5;
+    border-radius: 15px;
+    -moz-border-radius: 15px;
+    -webkit-border-radius: 15px;
+  }
+
+  input[type="submit"] {
```

Comment:
- [ ] As suggested by the previous reviewer, I kindly suggest that you add the button interactions for active (when it's pressed) as shown in the Figma template below:

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on the pressed state has the `background: #2230D2;`

### EXAMPLE:
**for pressed state**
```
input[type="submit"]:active {
   background-color: #2230D2;
}
```